src/extensions/stores/cluster-store.ts

```diff
@@ -0,0 +1,121 @@
+import { clusterStore as internalClusterStore, ClusterId } from "../../common/cluster-store";
+import type { ClusterModel } from "../../common/cluster-store";
+import { Cluster } from "../../main/cluster";
+import { Singleton } from "../core-api/utils";
+
+export { Cluster } from "../../main/cluster";
+export type { ClusterModel, ClusterId } from "../../common/cluster-store";
+
+/**
+ * Store for all added clusters
+ */
+export class ClusterStore extends Singleton {
+
+  /**
+   * Active cluster id
+   */
+  activeClusterId = internalClusterStore.activeCluster;
```

Comment:
have you tested that this works? When I was thinking about it yesterday I wasn't convinced that this would work and that we would have to do something like the following:

```suggestion
  get activeClusterId() {
    return internalClusterStore.activeCluster;
  }
  
  set activeClusterId(newId: string) {
      internalClusterStore.activeCluster = newId;
  }
```

Reply:
get/set is better.